package.json

```diff
@@ -216,59 +239,59 @@
         },
```

Comment:
I didn't add `magit.help` to the command palette as it I don't see a way it'd be useful (and it's the same as `magit.dispatch` anyways).